tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -48,18 +48,18 @@ using tensorflow::tracing::ScopedAnnotation;
 // Implementation note: HLO profiling is always enabled for GPU executables,
 // since we can use timers around thunks.
 GpuExecutable::GpuExecutable(
-    const string& ptx, const std::vector<uint8>& cubin,
-    std::pair<int, int> compute_capability,
+    const string& text, const std::vector<uint8>& binary,
+//    std::pair<int, int> compute_capability,
```

Comment:
@sumesh13 could you explain why we need to remove `compute_capability` here. I dislike this field very much as well but do we have to remove it?